analytic_engine/src/table_meta_set_impl.rs

```diff
@@ -192,6 +193,8 @@ impl TableMetaSetImpl {
                 };
                 self.find_space_and_apply_edit(space_id, alter_option)
             }
+            // no need to update memory.
+            MetaUpdate::AllocSstId(_) => Ok(()),
```

Comment:
I guess we can introduce a field called `max_file_id` in the `table_data` to make the logic here is uniform with other branches. And the `table_data.max_file_id` is only used to do manifest snapshot.